xrchat/xrchat-client
# Quick video pagination workaround

We don't have proper pagination for the video grid, so this is a quick workaround that just gets all 100 on page load.

What is the purpose of this PR? 
A quick workaround to get all 100 of Kai's videos in the Explore page

What does issue #s are solved by this PR?
No specific issue

What unit tests have been put in place?
None

How does our QA team test this PR?
Go to Explore page with ~100 videos in the database and make sure they are all in the grid.

What files or features are including in this PR that are not related to the main feature?
None
